website/static/js/koHelpers.js

```diff
@@ -207,13 +207,15 @@ ko.bindingHandlers.fadeVisible = {
     }
 };
 
+var tooltip = function(el, valueAccessor) {
+    var params = valueAccessor();
+    $(el).tooltip(params);
+};
 // Run Bootstrap tooltip JS automagically
 // http://getbootstrap.com/javascript/#tooltips
 ko.bindingHandlers.tooltip = {
-    init: function(el, valueAccessor) {
-        var params = valueAccessor();
-        $(el).tooltip(params);
-    }
+    init: tooltip,
+    update: tooltip
```

Comment:
@abought I would think any time the tooltip value is dependent on an observable. One use case:
- a button that when disabled has a tooltip helping to explain why, and a different caption when enabled.

That said, we can likely discard the init call.